src/utils/barcode.ts

```diff
@@ -0,0 +1,69 @@
+import * as bwipjs from "bwip-js";
+import { sequenceS } from "fp-ts/lib/Apply";
+import { Either, left, right, tryCatch2v } from "fp-ts/lib/Either";
+import { fromNullable } from "fp-ts/lib/Option";
+import { fromEither, taskEither, taskify } from "fp-ts/lib/TaskEither";
+import { getRequiredENVVar } from "./container";
+
+const BCID = fromNullable(getRequiredENVVar("BARCODE_ALGO_ID")).getOrElse(
```

Comment:
if you use `getRequiredENVVar` and `BARCODE_ALGO_ID` is not defined the server won't start and it will throw an error. use `NonEmptyString.decode(process.env.BARCODE_ALGO_ID)` for optional environment variables instead